File: beispiele/passwortgenerator/passwortgenerator.php
<?php 
$laenge = 15;

$kleine_buchstaben = "abcdefghijklmnopqrstuvwxyz";
$grosse_buchstaben = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
$zahlen = "0123456789";
$symbole = "!@#$%^&*()_-+=?><[]";
    
$zusammengesetzt = $kleine_buchstaben . $grosse_buchstaben . $zahlen . $symbole;
$passwort = substr(str_shuffle($zusammengesetzt), 0, $laenge);
    
echo "Das Passwort lautet: " . $passwort;
?>

File: beispiele/passwortgenerator/passwortgenerator.py
import random

laenge = 15

kleine_buchstaben = "abcdefghijklmnopqrstuvwxyz"
grosse_buchstaben = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
zahlen = "0123456789"
symbole = "!@#$%^&*()_-+=?><[]"
zusammengesetzt = kleine_buchstaben + grosse_buchstaben + zahlen + symbole
passwort = ''.join(random.sample(zusammengesetzt, laenge))

print("Das Passwort lautet:", passwort)